routes/web.php

```diff
@@ -29,10 +32,19 @@
     return Inertia::render('Dashboard');
 })->middleware(['auth', 'verified'])->name('dashboard');
 
+// guest can access this
+Route::resource('articles', ArticleController::class)->only(['index', 'show']);
+Route::resource('reports', ReportController::class)->only(['index', 'show']);
+Route::resource('campaigns', CampaignController::class)->only(['index', 'show']);
+
```

Comment:
// Todo: add routes for landing page (about-us, contact. etc)

Reply:
done king